Resources/Prototypes/_Den/Store/categories.yml

```
@@ -0,0 +1 @@
+#VAMPIRE
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)